src/framing/mod.rs

```diff
@@ -317,7 +318,13 @@ impl MLSCiphertext {
             content: mls_ciphertext_content.content,
             signature: mls_ciphertext_content.signature,
         };
-        let credential = roster.get(sender_data.sender.as_usize()).unwrap();
+        let credential = match indexed_members.get(&sender_data.sender) {
+            Some(c) => c,
+            None => {
+                return Err(MLSCiphertextError::UnknownSender);
```

Comment:
Maybe include a test case that triggers this error, so that codecov is a little happier?

Reply:
Done. It's in the framing unit tests.